docker/mu-plugins/jetpack-debug-helper/inc/class-broken-token-xmlrpc.php

```diff
@@ -67,17 +92,39 @@ public function render_ui() {
 			<div id="current_xmlrpc_errors">
 
 
-				<form action="<?php echo esc_url( admin_url( 'admin-post.php' ) ); ?>" method="post">
+				<form action="<?php echo esc_attr( admin_url( 'admin-post.php' ) ); ?>" method="post">
 					<input type="hidden" name="action" value="clear_all_xmlrpc_errors">
 					<?php wp_nonce_field( 'clear-xmlrpc-errors' ); ?>
 					<h2>
 						Current Unverified Errors
 						<input type="submit" value="Clear all unverified errors" class="button button-primary">
 					</h2>
 				</form>
+				<p>
+					Unverified errors are errors that were detected but that we don't know if they are legit and came from WP.com
```

Comment:
Yes this is the jetpack-debug-helper plugin mu-plugins but shouldn't we be making these translatable strings __() esc_html__() etc

Reply:
I'm not terribly concerned with translatable strings. They won't make it to WordPress.org, so they won't hit translate.wordpress.org, so they won't be part of the Jetpack language packs.

For straight text like this, we don't need to escape it -- only if vars are thrown in.

Reply:
agree with Kraft